types/postcss-js/tsconfig.json

```diff
@@ -0,0 +1,16 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": ["es2018"],
```

Comment:
Is this the right lib level for this package? It looks like other postcss DT packages have `["es6", "ES2018.Promise"]` (see rtlcss, postcss-less, others).

Reply:
I just pushed a commit to change it